### ISSUE TITLE
sphinx-autobuild: error (docker-compose up fails)

### ISSUE THREAD

**Issue by bittner** [appuio/docs#81]
The Python package for autobuilding the documentation, [sphinx-autobuild](https://pypi.org/project/sphinx-autobuild/#history), had a breaking change with the latest version (2020.9.1 of 31-Aug-2020). This makes `docker-compose` fail:

```console
$ docker-compose up
Starting appuiodocs_sphinx_1 ...
Starting appuiodocs_sphinx_1 ... done
Attaching to appuiodocs_sphinx_1
sphinx_1  | usage: sphinx-autobuild [-h] [--port PORT] [--host HOST]
sphinx_1  |                         [--re-ignore RE_IGNORE] [--ignore IGNORE]
sphinx_1  |                         [--no-initial] [--open-browser] [--delay DELAY]
sphinx_1  |                         [--watch DIR] [--pre-build COMMAND] [--version]
sphinx_1  |                         sourcedir outdir [filenames ...]
sphinx_1  | sphinx-autobuild: error: unrecognized arguments: -r .git
```

One way to fix this is to downgrade the package with a simple change to [requirements.txt](https://github.com/appuio/docs/blob/master/requirements.txt#L3):

```patch
diff --git a/requirements.txt b/requirements.txt
index 3842f36..be551cc 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
 recommonmark
 sphinx
-sphinx-autobuild
+sphinx-autobuild<2020.9.1
 sphinx_rtd_theme
```

The more sustainable way is to [fix the `-r` parameter](https://github.com/appuio/docs/blob/master/Dockerfile#L9), which now has the long form `--re-ignore`:

```patch
index 9847560..269fc7a 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -6,6 +6,6 @@ WORKDIR /src
 ADD requirements.txt .
 RUN pip install -r requirements.txt

-CMD ["sphinx-autobuild", ".", "_build_html", "-r", ".git", "--host", "0.0.0.0"]
+CMD ["sphinx-autobuild", ".", "_build_html", "--re-ignore", ".git", "--host", "0.0.0.0"]

 # vim: ft=dockerfile
```

Do you care to have a PR for that?

**Comment by tobru** [comment]
> Do you care to have a PR for that?

That would be fantastic! Thank you very much.

**Comment by megian** [comment]
@bittner Thanks for bringing that up. Will you create the PR for this?

**Comment by megian** [comment]
Created the PR #85 .

**Comment by mhutter** [comment]
Fixed in #85, thanks for reporting!

**Comment by bittner** [comment]
Sorry, I didn't receive any of GitHub's notifications! Not sure why. Maybe GitHub's notification system was confused, because I was a member of the @appuio organization?

Glad you managed to fix it yourselves. :1st_place_medal:
